packages/esm-patient-chart-app/translations/en.json

```
@@ -18,8 +18,10 @@
   "causeOfDeath_title": "Cause of Death",
   "checkFilters": "Check the filters above",
   "close": "Close",
-  "closeWorkspaces_one": "Discard changes in {{count}} workspaces",
-  "closeWorkspaces_other": "Discard changes in {{count}} workspaces",
+  "closeAllOpenedWorkspaces_one": "Discard changes in {{count}} workspaces",
+  "closeAllOpenedWorkspaces_other": "Discard changes in {{count}} workspaces",
+  "closingAllWorkspacesPromptBody": "There are unsaved changes in the following workspaces. Do you want to discard changes in the following workspaces? {{workspaceNames}}",
```

It doesn't seem right to repeat the phrase 'following workspaces' twice. Also, wouldn't it be better to list out the open workspaces as list items?

```suggestion
  "closingAllWorkspacesPromptBody": "There are unsaved changes in several workspaces. Do you want to discard changes in the following workspaces? {{workspaceNames}}",
```